tsoa/lib/lib.w

```diff
@@ -0,0 +1,64 @@
+bring cloud;
+bring sim;
+bring ui;
+bring util;
+
+interface StartResponse {
+  inflight port(): num;
+  inflight close(): void;
+}
+
+struct SpecProps {
+  outputDirectory: str?;
+  specVersion: num?;
+}
+
+pub struct ServiceProps {
+  entryFile: str?;
```

Comment:
add some docstrings?

@yoav-steinberg @Chriscbr what are the prospects for docstring support?